releases/katello/3.16/settings

```diff
@@ -0,0 +1,4 @@
+FULLVERSION='3.16.0-rc4'
+FOREMAN_VERSION='2.1'
+OSES="rhel7 el8"
```

Comment:
I think this should be only rhel7 now

Reply:
I want to say i tried that, but still got errors on release pipelines.  I'll remove, shoudln't be hard to put back if it dies.